src/inc/eetwain.h

```diff
@@ -161,7 +161,7 @@ enum
 };
 
 #ifndef DACCESS_COMPILE
-
+#ifdef _TARGET_X86_
```

Comment:
Do you mean that this method is related with ``WIN64EXCEPTIONS`` (not ``_TARGET_X86_``)?